x-pack/plugins/apm/public/components/app/ServiceOverview/ServiceList/HealthBadge.tsx

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import React from 'react';
+import { EuiBadge } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import {
+  getSeverityColor,
+  Severity,
+} from '../../../../../common/anomaly_detection';
+import { useTheme } from '../../../../hooks/useTheme';
+
+export function HealthBadge({ severity }: { severity?: Severity }) {
+  const theme = useTheme();
+
+  let label: string = '';
+
+  switch (severity) {
```

Comment:
Perhaps extract this to `getSeverityLabel` and move it to `common/anomaly_detection`. That way it's aligned with `getSeverityColor`.